### DISCUSSION TITLE
I Can't Use Fooocus Memory Error

### DISCUSSION THREAD

**Discussion by heitorfg** [lllyasviel/Fooocus#3411] [Q&A] [answered]
Sorry for my ignorance on the subject, but I need your help.
Processor: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz 1.99 GHz
Installed RAM: 12GB
Graphics Card: GEFORCE MX150

First, I tried installing it on my D: drive, which is a 1TB HDD, but it gave the following error when I clicked on "generate image":
The instruction at 0x00007FF938685474 referenced memory at 0x000000000000000000. The memory could not be read.
Click "OK" to terminate the program.

After that, I installed it on my 120GB SSD, C: drive. It opens but closes by itself or gives a blue screen error, not allowing it to run. I noticed that the SSD memory is at 100% usage.

![image](https://github.com/user-attachments/assets/4309f5d5-24cc-40f9-b681-68c14fa693db)
![image](https://github.com/user-attachments/assets/aa660b02-73f3-460b-a917-5154f9459028)
![image](https://github.com/user-attachments/assets/26220afd-0cad-4ca4-9192-a823422849f8)
![erro azul](https://github.com/user-attachments/assets/94bbe544-0df2-4b89-a2f7-cbdf636ed2f1)

I set up Virtual Swap, which is marked as "automatic".
What can I do to make the program work?

**Comment by mashb1t** [comment, marked as answer]
You're not meeting minimal system requirements: https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement

min. VRAM for GTX 10xx (similar to MX150) is 6GB, you have 4GB. Also the GPU seems not to be supported anymore for GPU driver updates (EOL 2021/2022).

You may have more luck with using --always-cpu in your run.bat, but your RAM is insufficient for that.

 => Your Laptop is too old to run Fooocus, i'm sorry.

**Reply by heitorfg** [reply]
Maybe I'll be lucky. 

How do I change --always-cpu?

Can you show me how to do it?

**Reply by mashb1t** [reply]
See readme, add `--always-cpu` in the same line after entry_with_update.py in the run.bat

**Reply by heitorfg** [reply]
Unfortunately it didn't work out. Do you know of other similar software or websites that could help me?

**Reply by mashb1t** [reply]
You may use Fooocus in [Google Colab](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#colab).
